zachang/HelloBooks
# #149947977 users should be able to view the list of all books

What does this PR do?

It allows users view the list of all books.

Description of Task to be completed?

*A page is created where users can view all books available.

How should this be manually tested?

Users should login to view all books.
Any background context you want to provide?

none for now.

What are the relevant pivotal tracker stories?

#149947977

Screenshots

![image](https://user-images.githubusercontent.com/23665997/32799913-4c99bd1c-c979-11e7-9f20-69a386524dd0.png)

Questions:

none for now
